.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,34 @@
+name: Update IntelliJ Version
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
+      - name: Prepare variables for PR
+        # language=sh
+        run: |
+          release_line=$(grep 'release = \"' core/driver/sources/src/main/resources/reference.conf)
+          intellij_version=$(awk -F'\"|\"' '{print $2}' <<< "$release_line")
```

Comment:
Or even simpler
```suggestion
          intellij_version=$(cut -d'"' -f2 <<< "$release_line")
```

Reply:
For me both expressions are not simple to be honest (you have to study manuals of `awk` or `cut` to understand, why it gets the desired value from line...) 😄  But maybe you are right - the `cut` command looks less scary. Let me change that.

Reply:
For my exp, `cut` is way more familiar to shell users than `awk` ;)